### PR TITLE
docs: improve installation notes for MacOS

### PR DESCRIPTION
**Pull request by puntonim**
## Checklist:

- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [ ] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [x] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by david-caro** [comment]
@drjova I guess that you are better suited to review this than me :), as you use macos too and going through a similar process.

**Comment by chris-asl** [comment]
When I was setting up my installation and tried submitting a record, `plotextractor` required a system-wide `pdftotext` installation. From searching around, the solution was to install `poppler` with `brew`.

File: INSTALL.rst [hunk 1]
```diff
@@ -99,6 +131,7 @@ setuptools to latest too:
     (inspirehep)$ pip install --upgrade pip setuptools
     (inspirehep)$ cdvirtualenv src/inspirehep
     (inspirehep)$ pip install -r requirements.txt --pre --exists-action i
+    (inspirehep)$ pip install honcho
```
Comment (drjova) [inline INSTALL.rst]: Shall we add ``honcho`` as a dependency in the ``webserver``? https://github.com/inspirehep/inspire-next/blob/master/setup.py#L133 :)

**Comment by drjova** [comment]
I think we should also update the copyright header year too ``2017-2018`` :)

**Comment by puntonim** [comment]
All right, I'll test the submission and update the year

**Comment by puntonim** [comment]
ci test please

**Comment by puntonim** [comment]
ci test please

**Comment by drjova** [comment]
@puntonim could you please rebase? :shipit:

**Comment by puntonim** [comment]
Done

**Pull request merged**